### PR TITLE
Add hint text to form fields

### PR DESCRIPTION
**Pull request by derekprior**
The placeholder text wasn't enough of an example and some users found it
confusing, thinking the placeholder was default text.

### CODE REVIEW

**Comment by derekprior** [comment]
@mborsare the hints need some design love. @jdaries was thinking tooltips on focus, but I'm sure he's open to whatever ideas you've got.

**Comment by mborsare** [comment]
image

**Comment by mborsare** [comment]
@jdaries I kept this simple – thoughts?

**Comment by jdaries** [comment]
@mborsare Looks great!!

File: app/assets/stylesheets/base/_forms.scss [hunk 1]
```diff
@@ -90,3 +91,51 @@ select {
     }
   }
 }
+
+.field_with_hint {
```
Comment (ryancoughlin) [inline app/assets/stylesheets/base/_forms.scss]: Should these be dashes?

Comment (ryancoughlin) [inline app/assets/stylesheets/base/_forms.scss]: Or are you going off an existing code base?

**Comment by ryancoughlin** [comment]
@mborsare Looks good.

File: app/assets/stylesheets/base/_forms.scss (hunk 1 above)
Comment (mborsare) [inline app/assets/stylesheets/base/_forms.scss]: yup, it's a wild world in there – gonna leave it

**Comment by mborsare** [comment]
I'm going to merge this in unless you need to do anything else on it @derekprior

**Comment by mborsare** [comment]
fixed this up

image

**Comment by derekprior** [comment]
Great. Thanks!

**Pull request merged**